pennylane/qchem/vibrational/pes_generator.py

```diff
@@ -736,3 +743,116 @@ def pes_threemode(
         return pes_threebody, dipole_threebody
 
     return pes_threebody, None  # pragma: no cover
+
+
+def vibrational_pes(
+    molecule,
+    quad_order=9,
+    method="rhf",
+    localize=True,
+    loc_freqs=[2600],
+    cubic=False,
+    dipole_level=1,
+):
+    r"""Computes potential energy surfaces along vibrational normal modes.
+
+    Args:
+       molecule (~qchem.molecule.Molecule): Molecule object
+       quad_order (int): Order for Gauss-Hermite quadratures. Default value is ``9``.
+       method (str): Electronic structure method that can be either restricted and unrestricted
+                Hartree-Fock,  ``'rhf'`` and ``'uhf'``, respectively. Default is ``'rhf'``.
+       localize (bool): Whether to perform normal mode localization. Default is ``False``.
+       freqs (list[float]): Array of frequencies in ``cm^-1`` where separation happens for mode localization.
+       cubic (bool)): Whether to include three-mode couplings. Default is ``False``.
+       dipole_level (int): Defines the level up to which dipole matrix elements are to be calculated. Input values can be
+                     ``1``, ``2``, or ``3`` for up to one-mode dipole, two-mode dipole and three-mode dipole, respectively. Default
+                     value is ``1``.
+
+    Returns:
+       VibrationalPES object.
+
+    """
+    if dipole_level > 3 or dipole_level < 1:
+        raise ValueError(
+            "The method only supports calculation of one-mode, two-mode and three-mode dipoles. Please provide a value"
+            "between 1 and 3."
+        )
+    if quad_order < 1:
+        raise ValueError("Number of sample points cannot be less than 1.")
+
+    _import_mpi4py()
+    from mpi4py import MPI
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    molecule, scf_result = optimize_geometry(molecule, method)
+
+    freqs = None
+    uloc = None
+    displ_vecs = None
+    if rank == 0:
+        freqs, displ_vecs = _harmonic_analysis(scf_result, method)
```

Comment:
This will require declaring unnecessary variables for _harmonic_analysis and reassigning some of the already existing variables in _localize_normal_modes. Since the return name doesn't make much of a difference and these are private functions, we can avoid it here.